### ISSUE TITLE
Not exactly an issue more of a question

### ISSUE THREAD

**Issue by PillowEmperor** [Shayne55434/Tautulli2Discord#1]
So i've got 3 of the scripts scheduled right now(not sure yet if they trigger), but there is no way to have it replace the message rather than placing a new one right? cause i'd like it to update every 5 minutes but it's quite alot of messages if it makes a new message.

**Comment by FSCorrupt** [comment]
Have a look at autodelete bot, you can set on a specified channel max message number (1) and he automatically deletes the old one, kinda replace!

**Comment by PillowEmperor** [comment]
> Have a look at autodelete bot, you can set on a specified channel max message number (1) and he automatically deletes the old one, kinda replace!

Will look into this thanks, somehow the script is currently still refusing to run with task schedule so don't need it quite yet haha.

**Comment by Shayne55434** [comment]
> edit2: Runs now, though for some reason still can't grab storage data for the other libraries.

Ooops. I used to have instructions in the PowerShell script for this one. You need to go into Tautulli > Settings > General > and enable "Calculate Total File Sizes". It may take a while for Tautulli to update the stats, depending on your library sizes.

Also, I can't seem to get it to calculate properly for my Music library. Not sure if it's too big or what.

**Comment by PillowEmperor** [comment]
> > edit2: Runs now, though for some reason still can't grab storage data for the other libraries.
> 
> Ooops. I used to have instructions in the PowerShell script for this one. You need to go into Tautulli > Settings > General > and enable "Calculate Total File Sizes". It may take a while for Tautulli to update the stats, depending on your library sizes.
> 
> Also, I can't seem to get it to calculate properly for my Music library. Not sure if it's too big or what.

Thanks mate enabled that and will see if it will start indexing the others libraries aswell, by the way is there still gonna be upgrades to this like if the content is playing or paused? or will it stay like this (Just wondering i'm perfectly fine with running how it runs right now)

**Comment by Shayne55434** [comment]
> By the way is there still gonna be upgrades to this like if the content is playing or paused? or will it stay like this (Just wondering i'm perfectly fine with running how it runs right now)

I have mine set to run every 5 minutes. It won't update to state whether a stream is paused or playing, but it will update the percentage complete. Perhaps I could incorporate the stream state. Shouldn't be too difficult.

**Comment by Shayne55434** [comment]
> By the way is there still gonna be upgrades to this like if the content is playing or paused? or will it stay like this (Just wondering i'm perfectly fine with running how it runs right now)

Just updated the CurrentStreams script. It now includes the play state in the footer. Take a look, if you'd like.

**Comment by PillowEmperor** [comment]
> > By the way is there still gonna be upgrades to this like if the content is playing or paused? or will it stay like this (Just wondering i'm perfectly fine with running how it runs right now)
> 
> Just updated the CurrentStreams script. It now includes the play state in the footer. Take a look, if you'd like.

Will check it out!

**Comment by PillowEmperor** [comment]
Hm, so it worked for a while though it did still send said error but now it seems to refuse to send data at all. Whereas the old script still sends.

**Comment by Shayne55434** [comment]
> 
> 
> Hm, so it worked for a while though it did still send said error but now it seems to refuse to send data at all. Whereas the old script still sends.
> 
> after a while it picks up again though, it's a bit weird, I can't seem to find any specific reason behind it besides the errors I get when manually running it.

Judging by the error message, your metadata for that particular item does not include tmdb_id. That, or stream.guids only has 1 guid. I was going to build in a check for this data but, as my metadata all has tmdb info (as far as I'm aware), I didn't really bother.

You could try to refresh your metadata on items it fails on. I may also try to incorporate a bit more checking in the script. Like all of my scripts, this was a quick and dirty write up.

**Comment by Shayne55434** [comment]
> What's the time part in the embed for though cause it's not showing the current time for me nor is it anything like when it will finish or such.
> 
> Does this grab date / time from the machine to display when the message is from or does it use some other form of timezone?

Ah, that's my bad. My Tautulli instance refuses to display the correct time, despite being set to the correct TZ, so I had to improvise. You can simply replace the timestamp with this:

(Get-Date).ToString("yyyy-MM-ddTHH:mm:ss.Mss")

That should work for you. It simply shows the time the message was sent. Completely up to you to include it or not.

**Comment by PillowEmperor** [comment]
> > What's the time part in the embed for though cause it's not showing the current time for me nor is it anything like when it will finish or such.
> > Does this grab date / time from the machine to display when the message is from or does it use some other form of timezone?
> 
> Ah, that's my bad. My Tautulli instance refuses to display the correct time, despite being set to the correct TZ, so I had to improvise. You can simply replace the timestamp with this:
> 
> (Get-Date).ToString("yyyy-MM-ddTHH:mm:ss.Mss")
> 
> That should work for you. It simply shows the time the message was sent. Completely up to you to include it or not.

Yeah had already looked into it a little bit, mine seems to be incorrectly showing it aswell so i'll have to see if I can substract some hours instead of add, a quick google search should help me in getting that.

Thanks though I do appreciate that you've added the play / pause and the new embed looks nice.

**Comment by Shayne55434** [comment]
> Thanks though I do appreciate that you've added the play / pause and the new embed looks nice.
> 
> edit:
> 
> looks like it's as simple as adding a - before the number

Glad you got it working and that you like the updates. I just do it for fun and figured I might as well share with the community.

**Comment by PillowEmperor** [comment]
Well, ran it for a bit longer again now with every metadata changed to TMDB but it seems like it really doesn't like anime, even after force refresh of metadata and clearing it most anime I try to run breaks it from sending any data.
